crates/tx5/examples/mem-echo-stress.rs

```diff
@@ -0,0 +1,92 @@
+//! Mem backend timings. The hope is to have near ideal zero overhead.
+//! At time of writing, you can see we're pretty close to exactly
```

Comment:
What is proportional? 1 node, 33941 ops, then 10 x the number of nodes and 174041 ops which is about 5 x as many ops. Then again 10 x as many nodes with 175111 ops, roughly the same.

Regarding time/op it doubles from 1 to 10 nodes, then rises 10 fold as does the number of nodes, but that's the only proportionality I can make out.

Reply:
@jost-s - you got it exactly. 1 node is different because it hadn't yet maxed out my cpu usage. Once there are 10 nodes, my cpu is maxed and the time/op increases linearly with the number of nodes plus a little overhead.